docs/contributing-admins.rst

```diff
@@ -0,0 +1,57 @@
+:orphan:
```

Comment:
@Dreamsorcerer also, since you put a link to this doc from another doc, Sphinx won't consider it orphan anymore. So it's not really needed either way.

Reply:
I mentioned above, that's not what orphan means to Sphinx. It literally just means that it is not in a toctree, not that it is unreachable. So, it is needed in its current state.